site/templates/page_template.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This is a template used for the purpose of adding markdown content to the site, but it's not actually used by the site (i.e. during the compilation done by Render Engine) itself, correct? Can we move this to its own directory outside of `site`?

Reply:
I originally had it at root but moved it to templates to clean the org up. There could be a directory call `publish` or something

Reply:
Yeah, I'm good with something like that, just aiming to avoid content within `site` that's not actually required for the site build.